### ISSUE TITLE
Different height of panel on multi-monitor setup

### ISSUE THREAD

**Issue by RidgeA** [home-sweet-gnome/dash-to-panel#690]
Hello!

I am using the extension on dual-monitor setup and on different monitors it has different height
![2019-07-26_11-46](https://user-images.githubusercontent.com/7588876/61939531-24487c80-af9c-11e9-9be2-e6af98673ec4.png)

I suppose that this is because of "tray icons" that displays only on the primary monitor. 
Is it possible to adjust the panel height to make it equal?

**Comment by charlesg99** [comment]
Hi RidgeA, the tray icons shouldn't affect the panels height. What are your monitor resolutions/pixel densities? Are you using scaling of some sort?

The panels use a fixed pixel size, so if no scaling is involved, the only thing I can think of for them to have different heights would be the pixel density. Could you please provide screenshots of both of your complete panels without the red highlighting? I'd like to compare the app icon sizes.

Thanks!

**Comment by RidgeA** [comment]
>  What are your monitor resolutions/pixel densities? 

Both monitors are HP Z22n with 1920x1080 resolution. Not sure about pixel densities, however, I am pretty sure that it is equal for both.

> Are you using scaling of some sort?

No.

>  Could you please provide screenshots of both of your complete panels without the red highlighting?

sure
![Screenshot from 2019-07-29 10-50-42](https://user-images.githubusercontent.com/7588876/62030922-d24b6500-b1ee-11e9-965a-ea28144471a3.png)
![Screenshot from 2019-07-29 10-50-39](https://user-images.githubusercontent.com/7588876/62030923-d24b6500-b1ee-11e9-8c19-d0c45659ce28.png)


Also, I've tried to disable all extensions, that could affect the panel height - no success, the height is different.

Is there any took like a browser's dev-tool, that could show you paddings, margins, etc for Gnome?

**Comment by charlesg99** [comment]
Looks like a theming issue then, does it happen with a different theme? What theme is that? 

There is a tool to inspect gnome-shell, but it isn't as user friendly as the chrome dev-tools... It is called the "looking glass" and you can open it with <kbd>Alt</kbd>+<kbd>F2</kbd>, then enter the `lg` command. Thanks!

**Comment by RidgeA** [comment]
Apparently, this is a custom theme issue.
I've switched off `user-themes` extension and the height of the panel becomes equal.

Custom theme - Flat Remix Dark (https://github.com/daniruiz/Flat-Remix)

**Comment by charlesg99** [comment]
Ha I see, you're probably on v19 of Dash to Panel? We recently worked on flat-remix compatibility (see #633) and luckily, those improvements should be released very soon with [v20](https://github.com/home-sweet-gnome/dash-to-panel/releases/tag/v20) (as soon as the e.g.o. [review process](https://extensions.gnome.org/review/) is complete). Thanks!

**Comment by RidgeA** [comment]
Hmm, I switched from DashToDock after I had seen an article somewhere that v20 was released. 
I didn't know that there is some kind of review process for extensions...
Yes, I have 19th version installed now.

Also, I had an old version of flat-remix-gnome installed (manually).
I reinstalled theme from a repository, the choose `flat-remix-dark-fullpanel` theme ant the panel height became equal (with theme enabled).

Thank you and sorry for bothering.
